tests/output/t302-basis-f.out

```diff
@@ -1,11 +1,11 @@
-collograd[0]: -3.00000000  4.04508497 -1.54508497  0.50000000
-collograd[1]: -0.80901699  0.00000000  1.11803399 -0.30901699
-collograd[2]:  0.30901699 -1.11803399  0.00000000  0.80901699
-collograd[3]: -0.50000000  1.54508497 -4.04508497  3.00000000
-collograd[0]: -3.33200024  4.86015442 -2.10878235  0.58062817
-collograd[1]: -0.75755761 -0.38441439  1.47067023 -0.32869822
-collograd[2]:  0.32869822 -1.47067023  0.38441439  0.75755761
-collograd[3]: -0.58062817  2.10878235 -4.86015442  3.33200024
+collograd[0]:  3.00000000 -4.04508497  1.54508497 -0.50000000
+collograd[1]:  0.80901699  0.00000000 -1.11803399  0.30901699
+collograd[2]: -0.30901699  1.11803399  0.00000000 -0.80901699
+collograd[3]:  0.50000000 -1.54508497  4.04508497 -3.00000000
+collograd[0]: -2.97732648  3.61240959  0.42440915 -4.17292696
+collograd[1]: -0.88463145  0.06263227  0.56306894  1.37442322
+collograd[2]:  0.34431271 -1.52560211  0.49593798  0.54828267
+collograd[3]:  0.13546013 -0.41042147  0.25437347 -6.26545429
```

Comment:
Maybe I missed it, but why these changes?

Reply:
There was a divide by zero showing up, so I had to tweak the simultaneous diagonalization to make it more stable.

Reply:
Added a test and reverted this change. The symmetry of BP3 was indeed masking an issue.

Reply:
Thanks.